modules/installing-trustyai-service.adoc

```diff
@@ -0,0 +1,23 @@
+:_module-type: CONCEPT
+
+[id='installing-trustyai-service_{context}']
+= Installing the TrustyAI service for a data science project
+
+[role='_abstract']
+
+For each data science project (namespace) that contains models for which the data scientists on your team want to monitor, you need to install an instance of the TrustyAI service. This action installs TrustyAI into your project and provides access to its features for all models deployed within that project.
+
+ifdef::upstream[]
+You can install an instance of the TrustyAI service by using the {productname-short} dashboard or by using the OpenShift command-line interface (CLI).
+endif::[]
+
+[NOTE]
+====
+You should install only one instance of the TrustyAI service in a project. The presence of multiple instances in the same project can result in unexpected behavior.
```

Comment:
```suggestion
Install only one instance of the TrustyAI service in a project. Multiple instances in the same project can result in unexpected behavior.
```
I kind of feel like the second sentence is implied by the first, so I also suggest:
```suggestion
Install only one instance of the TrustyAI service in a project.
```